site/app/templates/course/CourseMaterials.twig

```diff
@@ -91,6 +92,16 @@
                <a onclick='newDeleteCourseMaterialForm("{{ path | url_encode }}", "{{ dir }}");'> <i class="fas fa-trash" aria-hidden="true" style="font-size: 16px; margin: 5px;"></i></a>
                 <label for="latest-events">Share to student on</label>
                 <input name="release_date" id="date_to_release_{{ id }}" class="date_picker" type="text" size="26"  value="{{ fileReleaseDates[path]}}"/>
+                {% if not(this_file_section is null) %}
+                <label>Sections: 
+                {% for key,value in this_file_section %}
```

Comment:
You can use the twig `join` command instead of this loop 
https://twig.symfony.com/doc/2.x/filters/join.html